tests/Fixtures/e2e/Example_Test/phpunit.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit backupGlobals="false"
+         backupStaticAttributes="false"
+         bootstrap="./vendor/autoload.php"
+         colors="true"
+         convertErrorsToExceptions="true"
+         convertNoticesToExceptions="true"
+         convertWarningsToExceptions="true"
+         processIsolation="false"
+         stopOnFailure="false"
+         syntaxCheck="false"
+>
+    <testsuites>
+        <testsuite name="Test Suite">
+            <directory>./tests/</directory>
```

Comment:
in composer.json `"Namespace_\\Test\\": "test/"`
but `tests` here

does it work?

Reply:
For some reason it does, probably because phpunit deals with loading the tests, rather than using composer for that.

I fixed it to tests though to avoid confusion